### ISSUE TITLE
comments for mini project2

### ISSUE THREAD

**Issue by SandyHao** [yxiao1996/EC601#2]
To start with, I am sorry to give the review that late, but it really took me a long time to go over all your files.

0.You did great job in both building models and writing thoroughly readme file. I found that you have tried "COCO", "VGG", "yolo" and in different folders you train the model to classify different objects. Besides, for different program, there are all detailed instruction for running the program or for setting the format of datasets.

1.In "recoginistion" folder, there are three kinds of model using pre-trained weights. In another folder for images, you print out the loss and accuracy for different. That is wonderful to have the comparation. But it is a pity that there is no image test file. A test file which allow users to test their own images using the weight file will be even more fantastic.

2.There are a lot of parameters in deep learning programs. Such as "model.add(Dropout(0.5))","train_datagen = ImageDataGenerator(rescale=1. / 255, shear_range=0.2,zoom_range=0.2, horizontal_flip=True)", maybe comparing the same model with different parameters then found out the best suit can give out other great conclusions.

3.Moreover, I have a problem. It seems that you use cat and dog in the program from folder"misc", using flowers from the program in folder "recoginistion", but use elephant and giraffe for the final work. So is that just the different of name? In fact you train all the photos? 

Thank you for your work. I have learned a lot from it.
